server/datastore/mysql/migrations/tables/20211202092042_RemovePolicyHistory.go

```diff
@@ -0,0 +1,57 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20211202092042, Down_20211202092042)
+}
+
+func Up_20211202092042(tx *sql.Tx) error {
+	_, err := tx.Exec("DROP VIEW IF EXISTS policy_membership")
+	if err != nil {
+		return errors.Wrap(err, "drop view policy_membership")
+	}
+
+	policyMembershipTable := `
+		CREATE TABLE IF NOT EXISTS policy_membership (
+			policy_id INT UNSIGNED NOT NULL,
+			host_id int(10) UNSIGNED NOT NULL,
+			passes BOOL DEFAULT NULL,
+			created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+			updated_at timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+			PRIMARY KEY (policy_id,host_id),
+			FOREIGN KEY fk_policy_membership_policy_id (policy_id) REFERENCES policies(id) ON DELETE CASCADE,
+			FOREIGN KEY fk_policy_membership_host_id (host_id) REFERENCES hosts(id) ON DELETE CASCADE,
+			INDEX idx_policy_membership_passes (passes),
+			INDEX idx_policy_membership_policy_id (policy_id),
+			INDEX idx_policy_membership_host_id_passes (host_id, passes)
+		);
+	`
+
+	if _, err := tx.Exec(policyMembershipTable); err != nil {
+		return errors.Wrap(err, "create policy membership table")
+	}
+
+	if _, err := tx.Exec(`insert ignore into policy_membership 
```

Comment:
Why `ignore`?

Reply:
Because if it fails, I would rather it fails for some rows and not all, and then it'll later on self correct than having the whole migration process fail.

Reply:
Just curious why would it fail... (I'm more on the side of fail-early, but yeah there's a trade-off...)

Reply:
I suppose it shouldn't given that it has no fk, but given how other migrations have bitten us in the past, I wanted to play it safe.